packages/aragon-wrapper/src/index.js

```diff
@@ -1416,7 +1416,7 @@ export default class Aragon {
     // Failed to find a path
     return {
       direct: false,
-      transactions: []
+      path: []
```

Comment:
Maybe we should just return both arrays all the time; right now the API is:

- `direct: false`: includes `path`
- `direct: true`: includes `transactions`

Reply:
Done in 28099da; I've opted to ensure both arrays are returned and removed `direct`, as it no longer really serves a purpose (it never really did, and just added API bloat). Users can tell whether the intent basket is possible or not by checking if either of `path`, or `transactions` is non-empty.

This is a breaking change with an associated fix in https://github.com/aragon/aragon/pull/886.